ui/tests/e2e/indexui.spec.js

```diff
@@ -8,18 +8,18 @@ test.describe('Index Page UI Tests', () => {
   });
 
   test('Test if Left Navigation Panel is displayed', async ({ page }) => {
-    await expect(page.locator('[data-test=navigation]')).toBeVisible();
+    await expect(page.locator('[data-testid=navigation]')).toBeVisible();
```

Comment:
Can we use playwright built in test id instead?

https://playwright.dev/docs/locators#locate-by-test-id

Reply:
yeah please